src/web/server/app/tests/services/notif.test.ts

```diff
@@ -0,0 +1,13 @@
+import * as notif from '../../services/notif';
+import { client } from '../../utils/client';
+
+describe('notifications', () => {
+  beforeAll(() => {
+    client.execute = jest.fn();

```

Comment:
you probably don't want to mock the client since services directly interact with the database. It's how you test whether your queries are correct

Reply:
That  would be more suited for an integration test but this is just a unit test. I saw you and yufeng were testing directly with the DB for your tests but I think you should remove the junk data used for testing after to prevent cluttering the DB with irrelevant data.